qiskit_machine_learning/algorithms/__init__.py

```diff
@@ -104,6 +104,7 @@
     "MultiClassObjectiveFunction",
     "OneHotObjectiveFunction",
     "QSVC",
+    "PegasosQSVC",
```

Comment:
This should be added to the docstring classifier section above where QSVC is - i.e. around line 50 as that is what causes it to be included in the API ref documentation

Reply:
Added a reference in the API ref.